platform-sdk/docs/proposals/README.md

```diff
@@ -0,0 +1,150 @@
+errors should be caught during this phase.
+
+### Creating a Draft Proposal
+
+1. Create a new branch for the proposal.
+2. Create a subdirectory of `platform-sdk/docs/proposals/` with the name of the proposal.
+3. Copy the template `template.md` into the subdirectory and give it an appropriate name.
+4. Fill out the template with the details of the proposal, removing any irrelevant sections or adding new sections as
+   needed. Diagrams and other files may be included in the proposal directory and linked from the proposal.
+5. Create a draft PR with the proposal documents.
+6. Put the draft PR in the [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) board in
+   the `Draft` status.
+7. Solicit feedback from stakeholders and other design collaborators.
+8. Iterate on the proposal until it is ready for voting. All open comments should be resolved while in the draft stage.
+
+### Submitting a Proposal for Voting
+
+When the proposal is ready for voting, take the following steps:
+
+1. Ensure all comments are resolved.
+2. Mark the PR as `Ready for Review`
+3. Update the status of the PR in
+   the [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) board to `Voting`
+
+Requests for immaterial changes can be addressed by pushing additional commits to the existing proposal
+PR while in `Voting`. Examples of immaterial changes are spelling or grammar errors, wording changes that do not modify
+meaning or intent, and formatting changes. Examples of material changes include changes to behaviors or APIs, addition
+or removal of diagrams, etc. If any material changes are needed, the PR must be closed and moved to the `Superseded`
+status. A new proposal PR is prepared and voting is restarted. The `Superseded` PR must reference the new PR to
+create an auditable history.
+
+---
+
+## Voting on A Proposal
+
+Votes follow the [Apache Voting](https://www.apache.org/foundation/voting.html#expressing-votes-1-0-1-and-fractions)
+scheme. Votes are cast in comments on the PR on a range between -1 to -0, +0 to +1 with the following semantics:
+
+* -1: is a veto.
+* -0 >= x > -1: is a vote generally against the proposal.
+* +0 <= x < +1: is a vote generally in favor of the proposal.
+* +1: is a vote in favor of the proposal.
+
+The magnitude of the numeric value of the vote indicates the strength of the sentiment behind the vote.
+
++1 votes can also be expressed by approving the PR. -1 votes can be expressed by "Requesting Changes" on the PR and
+must be accompanied by a comment explaining the reason for the veto.
+
+---
+
+## Acceptance of A Proposal
+
+A proposal becomes `Accepted` when the following criteria have been fulfilled:
+
+1. The proposal has been in the Voting state for at least 3 business days. Business days are defined as SwirldsLabs
+   working days and excludes weekends and company holidays.
+2. The proposal must have at least three +1 votes from platform code owners.
```

Comment:
I had a discussion with @netopyr today regarding apis that might be used by other teams (services, smart contracts, ...). How do we handle that? I want that an api change in base is reviewed by that parties, too.

Reply:
All impacted parties should be coordinated with before the voting phase to ensure that the proposed API makes sense for all stakeholders. This is part of the Architectural Acceptance phase that is outside the scope of this process.